string robot_id
string cart_id
string station_id
string bound_mode
string action
string command_id
uint8 status
